common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -119,6 +119,13 @@ describe 'Problem', ->
         @problem.render(problem_content_default)
         expect(@problem.$('.problem-progress').html()).toEqual "(2/2 points)"
 
+    describe 'with valid status and and string containing an integer like "0" for detail', ->
```

Comment:
Nit: "and" is duplicated.

Please add a comment in the test case stating that this only fails on Chrome.